lib/oxidized/model/weos.rb

```diff
@@ -0,0 +1,22 @@
+class WEOS < Oxidized::Model
+
+  #Westell WEOS, works with Westell 8178G, Westell 8266G
+
+  prompt /^(\s[\w.@-]+[#>]\s?)$/
+
+  cmd :all do |cfg|
+    cfg.each_line.to_a[1..-2].join
+  end
+
+  cmd 'show running-config' do |cfg|
+    cfg
+  end
+
+  cfg :telnet do
+    username /login:/
+    password /assword:/
```

Comment:
you can use `/password:/i` then. 😉 
I gather these devices have both prompts then?